src/App.tsx

```diff
@@ -14,11 +15,45 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  goodsFromServer: string[];
+  showList:boolean;
+};
+
+class App extends React.Component {
+  state:State = {
+    goodsFromServer,
+    showList: false,
```

Comment:
variable with boolean types better named like `isListVisible` - it nice when it starts with `is`